src/components/common/CustomizedModalBlock.tsx

```diff
@@ -0,0 +1,29 @@
+import {Block, BlockProps} from 'baseui/block'
+import * as React from 'react'
+import {useStyletron} from 'baseui'
+
+const rowBlockProps: BlockProps = {
+  display: 'flex',
+  width: '100%',
+  marginTop: '1rem',
+};
+
+const CustomizedModalBlock = (props:BlockProps) =>{
+  const [css] = useStyletron()
+  return (
+    <Block display='flex' minWidth='100%'  paddingBottom={'1rem'} {...rowBlockProps}
```

Comment:
Her har du inline style, prop styles over, og override lenger ned. Gjør alt om til en av de, minwidth og width for eksempel gir ikke mye mening her heller